dashboard/test/models/course_offering_test.rb

```diff
@@ -437,6 +437,20 @@ class CourseOfferingTest < ActiveSupport::TestCase
     assert_equal facilitator_to_teacher_course_versions[@unit_facilitator_to_teacher.course_version.id][:units].keys, [@unit_facilitator_to_teacher.id]
   end
 
+  test 'query count for assignable_course_offerings' do
+    Unit.stubs(:should_cache?).returns true
+
+    assert_cached_queries(0) do
```

Comment:
Worth mentioning that this one used to make 135 queries and the facilitator one made 136 queries.